base/src/bit.rs

```diff
@@ -407,7 +407,9 @@ impl FloatEncoding for f64 {
     }
 }
 
+#[allow(clippy::approx_constant)]
```

Comment:
Duplicated attribute. Besides I think we can just substitute related constants such as `core::f64::const::LOG2_10` in the tests.